From e5291b4fb0d9adfc9da510f4acc7330d57e3e415 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Tue, 21 Jun 2022 10:13:48 +0200
Subject: [PATCH] test: drop the at_exit() coredump check

since we don't truncate the journal anymore.

(cherry picked from commit 5309b56505dfccf9111cb5fe6461047725429e79)

Related: #2087652
---
 test/units/testsuite-21.sh | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/test/units/testsuite-21.sh b/test/units/testsuite-21.sh
index 604bf145ca..053d571a90 100755
--- a/test/units/testsuite-21.sh
+++ b/test/units/testsuite-21.sh
@@ -8,17 +8,7 @@ set -o pipefail
 systemctl list-jobs | grep -F 'end.service' && SHUTDOWN_AT_EXIT=1 || SHUTDOWN_AT_EXIT=0
 
 at_exit() {
-    # "Safety net" - check for any coredumps which might have not caused dfuzzer
-    # to stop & return an error (we need to do this now before truncating the
-    # journal)
-    # TODO: check fo ASan/UBSan errors
-    local found_cd=0
-    while read -r exe; do
-        coredumctl info "$exe"
-        found_cd=1
-    done < <(coredumpctl -F COREDUMP_EXE | sort -u)
-    [[ $found_cd -eq 0 ]] || exit 1
-
+    set +e
     # We have to call the end.service explicitly even if it's specified on
     # the kernel cmdline via systemd.wants=end.service, since dfuzzer calls
     # org.freedesktop.systemd1.Manager.ClearJobs() which drops the service
